python/ray/util/sgd/v2/session.py

```diff
@@ -226,3 +256,54 @@ def train_func():
     """
     session = get_session()
     return session.world_rank
+
+
+def load_checkpoint() -> Optional[Dict]:
+    """Loads checkpoint data onto the worker.
```

Comment:
can you specify that this is the latest checkpoint.

Reply:
Added `return` doc.